<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Question_1Class</class>
 <widget class="QMainWindow" name="Question_1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Question_1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;Times New Roman&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="SelectFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>着色判断</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>展示结果</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>加载数据</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FileLine">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>641</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TextFile">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>641</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Question_1.qrc"/>
 </resources>
 <connections/>
</ui>
